### PR TITLE
[config] Add YANG alerting for override

### PR DESCRIPTION
**Pull request by wen587**
#### What I did
Add alerting for YANG validation when load_minigraph during override. This is to alert early if golden config is invalid which will breaks GCU feature. 
#### How I did it
Add alerting when `is_yang_config_validation_enabled` is not set during load_minigraph with override
#### How to verify it
Unit test

### CODE REVIEW

File: config/main.py [hunk 1]
```diff
@@ -1968,6 +1968,17 @@ def override_config_table(db, input_config_db, dry_run):
             validate_config_by_cm(cm, ns_config_input, "config_input")
             # Validate updated whole config
             validate_config_by_cm(cm, updated_config, "updated_config")
+        else:
+            cm = None
+            try:
+                cm = ConfigMgmt(configdb=config_db)
```
Comment (qiluo-msft) [inline config/main.py:1974]: Please add comment what do you verify?

File: config/main.py [hunk 2]
```diff
@@ -1968,6 +1968,17 @@ def override_config_table(db, input_config_db, dry_run):
...
+                cm.validateConfigData()
+            except Exception as ex:
+                log.log_warning("Failed to validate running config. Alerting: {}".format(ex))
+
+            # Validate updated whole config
+            if cm:
+                validate_config_by_cm_alerting(cm, updated_config, "updated_config")
```
Comment (qiluo-msft) [inline config/main.py:1981]: also what do you verified here?

**Comment by wen587** [comment]
Hi @vaibhavhd , this extra YANG validation would be happen during load_minigraph. Just want to make sure this doesn't impact warm reboot.

**Comment by wangxin** [comment]
Discussed with @wen587 offline about using log_err or log_warning. Considering check failure does not really mean something is broken and not working, use log_warning may be better.

**Review by qiluo-msft — APPROVED** [review]

**Comment by StormLiangMS** [comment]
HI @wen587 this one more like an enhancement, is that must to have for 202305?

**Comment by wen587** [comment]
Hi @StormLiangMS , the PR is conflict fixed in https://github.com/sonic-net/sonic-utilities/pull/3195.
I suggest to add to 202305. This will help us to find YANG issue in PROD by syslog without using GCU features.

**Pull request merged**